src/webui/package.json

```diff
@@ -39,7 +39,7 @@
     "lodash": "^4.17.13",
     "lodash.template": "^4.5.0",
     "js-yaml": "^3.13.1",
-    "webpack-dev-server": "^3.1.11",
+    "webpack-dev-server": "3.0.0",
```

Comment:
“^”?

Reply:
^3.0.0 still have this question:
![image](https://user-images.githubusercontent.com/35484733/61845831-6e1b5f00-aed7-11e9-92e4-c97973215bf4.png)